bundle/edu.gemini.spModel.core/src/main/scala/edu/gemini/spModel/core/Wavelength.scala

```diff
@@ -103,6 +70,10 @@ object Wavelength {
   implicit val WavelengthOrdering: scala.Ordering[Wavelength] =
     scala.Ordering.by(_.toNanometers)
 
-
+  /** @group Implicit Conversions */
+  implicit def fromLength(l: Length): Wavelength = {
```

Comment:
I'd rather that we add pimping to explicitly go from `Length`to `Wavelength` when desired (think `JavaConverters` instead of `JavaConversions`).  As you point out, this type of implicit conversion can be tricky.  It seems like we'd be losing a lot of the benefit of having an explicit `Wavelength` type.